linux/mcompile/keymap/keymap.h

```diff
@@ -1,67 +1,532 @@
+	0x4F, // L"K_NP1",				// &H61
+	0x50, // L"K_NP2",				// &H62
+	0x51, // L"K_NP3",				// &H63
+	0x4B, // L"K_NP4",				// &H64
+	0x4C, // L"K_NP5",				// &H65
+	0x4D, // L"K_NP6",				// &H66
+	0x47, // L"K_NP7",				// &H67
+	0x48, // L"K_NP8",				// &H68
+	0x49, // L"K_NP9",				// &H69
+	0x37, // L"K_NPSTAR",			// &H6A
+	0x4E, // L"K_NPPLUS",			// &H6B
+	0x7E, // L"K_SEPARATOR",	// &H6C		// MCD 01-11-02: Brazilian Fix, 00 -> 7E
+	0x4A, // L"K_NPMINUS",		// &H6D
+	0x53, // L"K_NPDOT",			// &H6E
+	0x135, // L"K_NPSLASH",		// &H6F
+	0x3B, // L"K_F1",					// &H70
+	0x3C, // L"K_F2",					// &H71
+	0x3D, // L"K_F3",					// &H72
+	0x3E, // L"K_F4",					// &H73
+	0x3F, // L"K_F5",					// &H74
+	0x40, // L"K_F6",					// &H75
+	0x41, // L"K_F7",					// &H76
+	0x42, // L"K_F8",					// &H77
+	0x43, // L"K_F9",					// &H78
+	0x44, // L"K_F10",				// &H79
+	0x57, // L"K_F11",				// &H7A
+	0x58, // L"K_F12",				// &H7B
+	0x64, // L"K_F13",				// &H7C
+	0x65, // L"K_F14",				// &H7D
+	0x66, // L"K_F15",				// &H7E
+	0x67, // L"K_F16",				// &H7F
+	0x68, // L"K_F17",				// &H80
+	0x69, // L"K_F18",				// &H81
+	0x6A, // L"K_F19",				// &H82
+	0x6B, // L"K_F20",				// &H83
+	0x6C, // L"K_F21",				// &H84
+	0x6D, // L"K_F22",				// &H85
+	0x6E, // L"K_F23",				// &H86
+	0x76, // L"K_F24",				// &H87
+
+	0x00, // L"K_?88",				// &H88
+	0x00, // L"K_?89",				// &H89
+	0x00, // L"K_?8A",				// &H8A
+	0x00, // L"K_?8B",				// &H8B
+	0x00, // L"K_?8C",				// &H8C
+	0x00, // L"K_?8D",				// &H8D
+	0x00, // L"K_?8E",				// &H8E
+	0x00, // L"K_?8F",				// &H8F
+
+	0x45, // L"K_NUMLOCK",		// &H90
+	0x46, // L"K_SCROL0x00, 	// &H91
+
+	0x00, // L"K_?92",				// &H92
+	0x00, // L"K_?93",				// &H93
+	0x00, // L"K_?94",				// &H94
+	0x00, // L"K_?95",				// &H95
+	0x00, // L"K_?96",				// &H96
+	0x00, // L"K_?97",				// &H97
+	0x00, // L"K_?98",				// &H98
+	0x00, // L"K_?99",				// &H99
+	0x00, // L"K_?9A",				// &H9A
+	0x00, // L"K_?9B",				// &H9B
+	0x00, // L"K_?9C",				// &H9C
+	0x00, // L"K_?9D",				// &H9D
+	0x00, // L"K_?9E",				// &H9E
+	0x00, // L"K_?9F",				// &H9F
+	0x2A, // L"K_?A0",				// &HA0
+	0x36, // L"K_?A1",				// &HA1
+	0x1D, // L"K_?A2",				// &HA2
+	0x1D, // L"K_?A3",				// &HA3
+	0x38, // L"K_?A4",				// &HA4
+	0x38, // L"K_?A5",				// &HA5
+	0x6A, // L"K_?A6",				// &HA6
+	0x69, // L"K_?A7",				// &HA7
+	0x67, // L"K_?A8",				// &HA8
+	0x68, // L"K_?A9",				// &HA9
+	0x65, // L"K_?AA",				// &HAA
+	0x66, // L"K_?AB",				// &HAB
+	0x32, // L"K_?AC",				// &HAC
+	0x20, // L"K_?AD",				// &HAD
+	0x2E, // L"K_?AE",				// &HAE
+	0x30, // L"K_?AF",				// &HAF
+	0x19, // L"K_?B0",				// &HB0
+	0x10, // L"K_?B1",				// &HB1
+	0x24, // L"K_?B2",				// &HB2
+	0x22, // L"K_?B3",				// &HB3
+	0x6C, // L"K_?B4",				// &HB4
+	0x6D, // L"K_?B5",				// &HB5
+	0x6B, // L"K_?B6",				// &HB6
+	0x21, // L"K_?B7",				// &HB7
+	0x00, // L"K_?B8",				// &HB8
+	0x00, // L"K_?B9",				// &HB9
+	0x27, // L"K_COLON",			// &HBA
+	0x0D, // L"K_EQUA0x00, // L",	// &HBB
```

Comment:
Looks like this ocmment had an accident :smiley:

Reply:
..and 2 others as well:
```
windows/src/global/cpp/VKScanCodes.cpp
windows/src/engine/keyman32/VKScanCodes.cpp
```